.github/actions/check-skip/action.yaml

```diff
@@ -0,0 +1,144 @@
+name: 'Check skips'
+description: 'Check whether some workflows should be skipped'
+inputs:
+  ignore-changes:
+    required: false
+    description: |
+      Directories and files to ignore when checking changes to determine whether functional
+      tests should be skipped. This is an optional input field and it accepts a regex string.
+      The paths should be relative to the repo's root directory.
+      Examples:
+      - "^(README.md|doc/|tests/unit/)"
+    default: "^$"
+  check-lint-unit-skip:
+    required: false
+    description: "Whether to check the possibility of skipping lint and unit tests"
+    default: false
+  check-func-skip:
+    required: false
+    description: "Whether to check the possibility of skipping functional tests"
+    default: false
```

Comment:
I do not think that this inputs are needed.

Reply:
They were needed because I was calling check-skip for func workflow and lint-unit workflow separately. Since now we decided to move the check to `pull-request.yaml`, these inputs are no longer needed. I removed them in the latest commit.